### PR TITLE
[docs] Fix article text in «Cloud provider — GCP: Preparing environment» page

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

The "Cloud provider — GCP: Preparing environment" article reviewing and fixing.

## Changelog entries

```changes
module: docs
type: fix
description: "Review and fix the 'Cloud provider — GCP: Preparing environment' article."
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md [hunk 1]
```diff
@@ -2,64 +2,69 @@
...
+```
+3\. Создайте service account:
+
+```shell
+gcloud iam service-accounts create $SERVICE_ACCOUNT_NAME
+```
+4\. Присвойте роли созданному service account:
+
+```shell
+for role in roles/compute.admin roles/iam.serviceAccountUser roles/networkmanagement.admin; do gcloud projects add-iam-policy-binding ${PROJECT} --member=serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT}.iam.gserviceaccount.com --role=${role}; done
```
Comment (z9r5) [inline candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md:49]:
```suggestion
for role in roles/compute.admin roles/iam.serviceAccountUser roles/networkmanagement.admin; 
do gcloud projects add-iam-policy-binding ${PROJECT} --member=serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT}.iam.gserviceaccount.com --role=${role}; done
```

File: candi/cloud-providers/gcp/docs/ENVIRONMENT.md [hunk 2]
```diff
@@ -19,47 +21,49 @@ Network Management Admin
...
+```
+3\. Create a service account:
+
+```shell
+gcloud iam service-accounts create $SERVICE_ACCOUNT_NAME
+```
+4\. Connect roles to the service account:
+
+```shell
+for role in roles/compute.admin roles/iam.serviceAccountUser roles/networkmanagement.admin; do gcloud projects add-iam-policy-binding ${PROJECT} --member=serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT}.iam.gserviceaccount.com --role=${role}; done
```
Comment (z9r5) [inline candi/cloud-providers/gcp/docs/ENVIRONMENT.md:49]:
```suggestion
for role in roles/compute.admin roles/iam.serviceAccountUser roles/networkmanagement.admin; 
do gcloud projects add-iam-policy-binding ${PROJECT} --member=serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT}.iam.gserviceaccount.com --role=${role}; done
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — CHANGES_REQUESTED** [review]
@Zhbert Please review my changes

File: candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md [hunk 3]
```diff
@@ -2,24 +2,22 @@
...
 
 > **Внимание!** Созданный `service account key` невозможно восстановить, только удалить и создать новый.
 
-Далее представлена краткая последовательность необходимых действий.
+## Настройка через Google Cloud Console
 
-## Настройка через Google cloud console
+Перейдите по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый service account (также можно выбрать уже существующий).
 
-Перейдите по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый сервисный аккаунт (также можно выбрать уже существующий).
```
Comment (Zhbert) [inline candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md]:
```suggestion
Перейдите [по ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый сервисный аккаунт (также можно выбрать уже существующий).
```
I think it's more correct. The preposition should also be included in the link.

**Review by Zhbert — COMMENTED** [review]
Everything is fine, I found only my own mistake.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md [hunk 4]
```diff
@@ -2,64 +2,68 @@
...
+Для того чтобы Deckhouse мог управлять ресурсами, в Google Cloud необходимо создать service account. Далее представлена краткая последовательность действий по созданию service account. Если вам необходима более подробная инструкция, вы можете найти ее [в документации провайдера](https://cloud.google.com/iam/docs/service-accounts).
 
-> **Внимание!** `service account key` невозможно восстановить, только удалить и создать новый.
+> **Внимание!** Созданный `service account key` невозможно восстановить, только удалить и создать новый.
 
-## Настройка через Google cloud console
+## Настройка через Google Cloud Console
 
-Переходим по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts) , выбираем проект и создаем новый сервис-аккаунт или выбираем существующий.
+Перейдите по [ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый service account (также можно выбрать уже существующий).
```
Comment (z9r5) [inline candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md:11]:
```suggestion
Перейдите [по ссылке](https://console.cloud.google.com/iam-admin/serviceaccounts), выберите проект и создайте новый service account (также можно выбрать уже существующий).
```

File: candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md (hunk 3 above)
Comment (z9r5) [reply candi/cloud-providers/gcp/docs/ENVIRONMENT_RU.md]: fixed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**